assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -104,6 +110,62 @@ export const selectors = {
 		} );
 	} ),
 
+	/**
+	 * Gets the Site Kit documentation URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {string} slug  The slug of the documentation page.
+	 * @return {string} The Site Kit support URL.
+	 */
+	getDocumentationLinkURL: createRegistrySelector(
+		( select ) => ( state, slug ) => {
+			invariant( slug, 'A slug is required.' );
+
+			const proxySupportLink = select(
+				CORE_SITE
+			).getProxySupportLinkURL();
+
+			return `${ proxySupportLink }?doc=${ encodeURIComponent( slug ) }`;
+		}
+	),
+
+	/**
+	 * Gets the relevant troubleshooting URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @param {Object} error The error object.
+	 * @return {string} The Site Kit support URL.
+	 */
+	getErrorTroubleshootingLinkURL: createRegistrySelector(
+		( select ) => ( state, error ) => {
+			invariant( error, 'An error is required.' );
+
+			const proxySupportLink = select(
+				CORE_SITE
+			).getProxySupportLinkURL();
+
+			if ( error.id && ! isNumber( error.id ) ) {
+				return `${ proxySupportLink }?error_id=${ encodeURIComponent(
+					error.id
+				) }`;
+			}
+
+			if ( error.code && ! isNumber( error.code ) ) {
+				return `${ proxySupportLink }?error_id=${ encodeURIComponent(
+					error.code
+				) }`;
+			}
```

Comment:
As mentioned in the test code, I think the `isNumber` check here isn't quite strict enough; the ACs are looking for a string that isn't solely numeric.